### DISCUSSION TITLE
difference between ha core backup and an auto backup full?

### DISCUSSION THREAD

**Discussion by kinghat** [jcwillox/hass-auto-backup#131] [Q&A] [answered]
looking at the daily auto backup and the regular ha core backup that is made when upgrading ha versions, the former is about 3x the size:

![image](https://github.com/jcwillox/hass-auto-backup/assets/16073360/06a8649d-7c73-45e6-9a8d-37638de28af1)

**Comment by jcwillox** [comment, marked as answer]
The core backup only includes your home assistant folder so you can rollback to the previous version of home assistant, the fully backup additionally includes all the add-ons and all the other folders like media.

Selecting the backup should show you what it includes.

Home Assistant is the one that handles the backup process, so it will be identical, really the main part that is configurable is what folders or add-ons are backed up.
